code/API_definitions/Call_Forwarding_Signal v0.1.yml

```diff
@@ -79,26 +79,31 @@ paths:
         - openId:
           - call-forwarding:read
       description: Retrieve the information about call forwarding service active on a phone number     
-      operationId: retriveCallForwarding
+      operationId: retrieveCallForwarding
+      parameters:
+        - $ref: '#/components/parameters/x-correlator'
       requestBody:
         content:
           application/json:
             schema:
-              $ref: '#/components/schemas/PhoneNumber'
+              $ref: '#/components/schemas/RequestPhoneNumber'

```

Comment:
@bigludo7 why "Request".

I suggest to define the variable with no relation to the action. Just as generic "phone number". Currently it is an input, maybe it could be, in some future release, even an output.

In the other CAMARA API, we define just PhoneNumber. Usually it is part of Device. An anyway, if it is Device, it is just "Device" and not "RequestDevice". Is there some guideline I'm missing?

Reply:
We can change the name of the schema, no problem.

My point was to have in the request a structure like:
```
{
"phoneNumber": "+33123456789"
}
```

and not

```
{
"+33123456789"
}
```

In order to be consistent with all other CAMARA API.